keyboards/ut472/info.json

```diff
@@ -5,6 +5,57 @@
   "width": 12.5,
   "height": 4,
   "layouts": {
+    "LAYOUT_ut472": {
```

Comment:
This block isn't needed, if there is no layout macro to match it.  And having it present may cause issues for the Configurator.

Reply:
Removing this now.